tools/qontract_cli.py

```diff
@@ -626,7 +627,8 @@ def sre_checkpoints(ctx):
     print_output(ctx.obj['options'], checkpoints_data, columns)
 
 
-def print_output(options, content, columns=[]):
+def print_output(options: Mapping[str, Union[str, bool]],
+                 content: List[Dict], columns: Iterable[str] = []):
```

Comment:
```suggestion
                 content: List[Dict], columns: Iterable[str] = ()):
```
don't use a mutable value as a default. Empty tuples usually work just as well.